.github/workflows/enteprise.yml

```diff
@@ -173,62 +175,52 @@ jobs:
           # avoid auth conflict when cloning DTAS during verify step
           persist-credentials: false
 
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        id: setup-python
+      - name: Run tests on push
+        timeout-minutes: 185
+        uses: ./.github/actions/ec2
         with:
-          python-version-file: '.python-version'
-          cache: pipenv
-
-      - name: Install python packages
-        uses: Alfresco/alfresco-build-tools/.github/actions/pipenv@v5.22.3
-        with:
-          python-version: ${{ steps.setup-python.outputs.python-version }}
-
-      - name: Install and cache ansible galaxy dependencies
-        uses: ./.github/actions/galaxy
-        with:
-          cache-name: enterprise
-
-      - name: Set BRANCH_NAME
-        uses: Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v5.22.3
+          matrix_name: ${{ matrix.molecule_scenario.name }}
+          matrix_vars: ${{ matrix.molecule_scenario.vars }}
+          matrix_desc: ${{ matrix.molecule_scenario.desc }}
+          nexus_username: ${{ secrets.NEXUS_USERNAME }}
+          nexus_password: ${{ secrets.NEXUS_PASSWORD }}
+          aws_access_key_id: ${{ secrets.aws_access_key_id }}
+          aws_secret_access_key: ${{ secrets.aws_secret_access_key }}
+          BOT_GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
+
+  ec2-manual:
```

Comment:
don't reference the event name, now that I see it I think it can be an idea to run this "extended tests" on pull_request merge and/or on a weekly schedule and/or applying manually a label on an open PR 🤔